homeassistant/components/zha/config_flow.py

```diff
@@ -155,7 +155,7 @@ async def async_step_confirm(self, user_input=None):
             )
 
         return self.async_show_form(
-            step_id="confirm",
+            step_id="confirm_usb",
```

Comment:
We didn't update the strings.json file for the changed step name.

Reply:
Addressed in #77010